shared/build.gradle.kts

```
@@ -42,20 +47,100 @@ kotlin {
         val commonMain by getting {
             dependencies {
                 implementation(Dependencies.Koin.CORE)
+
+                with(Dependencies.Ktor) {
+                    implementation(CORE)
+                    implementation(SERIALIZATION)
+                    implementation(LOGGING)
+                    implementation(CIO)
+                    implementation(CONTENT_NEGOTIATION)
+                    implementation(JSON)
+                    implementation(AUTH)
+                    implementation(JSON_API)
+                }
+
+                implementation(Dependencies.Log.NAPIER)
             }
         }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test"))
+                with(Dependencies.Test) {
+                    implementation(COROUTINES)
+                    implementation(KOTEST_ASSERTIONS)
+                    implementation(KOTLINX_RESOURCES)
+                    implementation(MOCKATIVE)
+                    implementation(TURBINE)
+                }
+            }
+        }
+        val androidUnitTest by getting
+        val androidMain by getting {
+            dependencies {
+                implementation(Dependencies.Koin.ANDROID)
+                implementation(Dependencies.Ktor.ANDROID)
```

still

```suggestion
                implementation(Dependencies.Koin.ANDROID)
```

---

@luongvo I need `Ktor` to declare module in `androidMain` for the following line:
```
single { Android.create() }
```

---

@kaungkhantsoe Ok, so no Koin?

```suggestion
                implementation(Dependencies.Ktor.ANDROID)
```

---

@kaungkhantsoe Have you checked that? ☝️ 👋

---

I need the Koin too. Following is the class content. 🙏 
```
import io.ktor.client.engine.android.Android
import org.koin.core.module.Module
import org.koin.dsl.module

actual fun platformModule(): Module = module {
    single { Android.create() }
}
```

---

@kaungkhantsoe it's already by `implementation(Dependencies.Koin.CORE)` from commandMain. `Koin.ANDROID` is for Android module instead of androidMain in the shared module. Please recheck 🤔

---

Removed in https://github.com/nimblehq/kmm-templates/pull/37/commits/d0922a65190a5f8d95b6cfc9f81a0237a9ebfa3f

---

@kaungkhantsoe please correct the commit message, to remove Koin, not Ktor 🙏

---

Ahh yes, I was confused 😆. Done @luongvo 👍